morphia/src/main/java/org/mongodb/morphia/query/FieldEndImpl.java

```diff
@@ -142,7 +142,13 @@ public T doesNotHaveThisElement(final Object val) {
     @Override
     public T hasThisElement(final Object val) {
         Assert.parametersNotNull("val", val);
-        return addCriteria(FilterOperator.ELEMENT_MATCH, val, false);
+        return addCriteria(FilterOperator.ELEMENT_MATCH, val, not);
```

Comment:
It is.  It's currently not possible to negate this operation because of that hard coded 'false'